docs/reference/README.md

```diff
@@ -1008,13 +1008,23 @@ This would match 'cat', 'caat', 'caaat', etc.
 
 ## reprompt
 
-Specifies the reprompt that will be installed during
-the next response. The content is specified in the
-[Say String](#say-string) format.
+Adds a [Say String](#say-string) to the pending skill response's reprompt speech.
+Same as the [say](#say) statement, any reprompted fragments are accumulated.
+```coffeescript
+reprompt "Hello"
+reprompt "World"
+# would result in Alexa reprompting "Hello World"
+```
+
+Alternatively, writing a standalone `reprompt` will add the entire pending
+response's `say` speech to the reprompt.
 
-Note: Unlike [say](#say) statements, reprompt statements
-are *not* accumulative; only the most recent reprompt statement
-will be included in the response.
+```coffeescript
+say "one"
+say "two"
+reprompt # would result in Alexa reprompting "one two three"
+say "three"
+```
```

Comment:
This was ultimately removed since it's relatively useless; would revert the changes for this PR.

Reply:
Looks like you might be addressing these in your say-reprompt syntax PR; I would merge that one first to make reverting this easier.